README.md

```diff
@@ -1 +1,207 @@
+- Download all the files from [this dir](/logan/docker-images/v1.0/debian/) into a local machine having access to internet.
+- Run the following command to build the docker image.
+    - *docker build -t fluentd_oci_la -f Dockerfile .*
+- The docker image built from above step, can be either pushed to Docker Hub or OCI Container Registry (OCIR) or to a Local Docker Registry depending on the requirements.
+    - [How to push the image to Docker Hub](https://docs.docker.com/docker-hub/repos/#pushing-a-docker-container-image-to-docker-hub)
+    - [How to push the image to OCIR](https://www.oracle.com/webfolder/technetwork/tutorials/obe/oci/registry/index.html).
+    - [How to push the image to Local Registry](https://docs.docker.com/registry/deploying/).
+    
+### Deploying Kuberenetes resources using Kubectl
+
+#### Pre-requisites
+
+- A machine having kubectl installed and setup to point to your Kubernetes environment.
+
+#### To enable Logs collection
+
+Download all the yaml files from [this dir](/logan/kubernetes-resources/logs-collection/).
+These yaml files needs to be applied using kubectl to create the necessary resources that enables the logs collection into Logging Analytics through a Fluentd based DaemonSet.
+
+##### configmap-docker.yaml | configmap-cri.yaml
+
+- This file contains the necessary out of the box fluentd configuration to collect Kubernetes System/Service Logs, Linux System Logs and Application Pod/Container Logs. 
+- Some log locations may differ for Kubernetes clusters other than OKE, EKS and may need modifications accordingly. 
+A comprehensive out of the box covering typical kubernetes clusters will be available soon.
+- Use configmap-docker.yaml for Kubernetes clusters based off Docker runtime (e.g., OKE < 1.20) and configmap-cri.yaml for Kubernetes clusters based off CRI-O.
+- Inline comments are available in the file for each of the source/filter/match blocks for easy reference for making any changes to the configuration.
+- Refer [this](https://docs.oracle.com/en/learn/oci_logging_analytics_fluentd/) to learn about each of the Logging Analytics Fluentd Output plugin configuration parameters.
+- *Note*: A generic source with time only parser is defined/configured for collecting all application pod logs from /var/log/containers/ out of the box. 
+          It is recommended to define and use a LogSource/LogParser at Logging Analytics for a given log type and then modify the configuration accordingly.
+          When adding a configuration (Source, Filter section) for any new container log, also exclude the log path from generic log collection, 
+            by adding the log path to *exclude_path* field in *in_tail_containerlogs* source block. This is to avoid the duplicate collection of logs through generic log collection.
+
+##### fluentd-daemonset.yaml
+
+- This file has all the necessary resources to deploy to run the Fluentd docker image as Daemonset.
```

Comment:
This file has all the necessary resources required to deploy and run the Fluentd docker image as Daemonset.